src/main/java/com/pwn9/filter/bukkit/BukkitAPI.java

```diff
@@ -63,6 +70,11 @@
 
     BukkitAPI(PwnFilterPlugin p) {
         plugin = p;
+        if(p instanceof PwnFilterBukkitPlugin) {
+            audiences = BukkitAudiences.create((PwnFilterBukkitPlugin) p);
+        } else {
+            audiences = null;
```

Comment:
Codacy found an issue: [Possible unsafe assignment to a non-final static field in a constructor.](https://app.codacy.com/gh/AddstarMC/PwnFilter/pullRequest?prid=6040651)